fiftyone/operators/utils.py

```
@@ -6,8 +6,9 @@
 |
 """
 
-from datetime import datetime
+from datetime import datetime, timedelta
```

Ruff

`datetime.timedelta` imported but unused

Remove unused import: `datetime.timedelta`

(F401)